File: 프로그래머스/0/120903. 배열의 유사도/배열의 유사도.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<string> s1, vector<string> s2) 
{
    int count = 0;
    for(string &s : s1)
    {
        for(string &t : s2)
        {
            if(s == t)
            {
                count++;
            }
        }
    }
    return count;
}

File: 프로그래머스/0/120903. 배열의 유사도/배열의 유사도.cs
using System;

public class Solution {
    public int solution(string[] s1, string[] s2) 
    {
        int count = 0;
        for(int i=0; i<s1.Length; i++)
        {
            for(int j=0; j<s2.Length; j++)
            {
                if(s1[i] == s2[j])
                {
                    count++;
                }
            }
        }
        return count;
    }
}